warning: typedef requires a name (for belt_item_type in ./SourceX/controls/plrctrls.h)

Compiling DevilutionX with Clang presents the following warning:

```
./SourceX/controls/plrctrls.h:8:1: warning: typedef requires a name [-Wmissing-declarations]
typedef enum belt_item_type {
^~~~~~~
```

--- a/SourceX/controls/plrctrls.h
+++ b/SourceX/controls/plrctrls.h
@@ -8,7 +8,7 @@ namespace dvl {
 typedef enum belt_item_type {
 	BLT_HEALING,
 	BLT_MANA,
-};
+} belt_item_type;
 
 // Run after every game logic iteration.
 // Handles player and menu movement.

